### PR TITLE
OmeroPy py3 update

### PR DESCRIPTION
**Pull request by will-moore**
In progress...

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-docs#91](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-docs/91/). See the [console output](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-docs/91/consoleText) for more details.
Possible conflicts:
  - PR #2010 will-moore 'Add annotation examples from saeri scripts'
    - omero/developers/Python.rst
  - PR #2011 will-moore 'Python example script links'
    - omero/developers/Python.rst

--conflicts

**Comment by joshmoore** [comment]
@will-moore : what do you want to do about #2010?

**Comment by will-moore** [comment]
I'd still like to get #2010 merged, so I'll list it for final review

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-docs#200](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-docs/200/). See the [console output](https://latest-ci.openmicroscopy.org/jenkins/job/OMERO-docs/200/consoleText) for more details.
Possible conflicts:
  - PR #2010 will-moore 'Add annotation examples from saeri scripts'
    - omero/developers/Python.rst
  - PR #2011 will-moore 'Python example script links'
    - omero/developers/Python.rst

~~--conflicts~~ Conflict resolved in build [OMERO-docs-superbuild-push#39](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs-superbuild-push/39/). See the [console output](https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs-superbuild-push/39/consoleText) for more details.

File: omero/developers/Python.rst [hunk 1]
```diff
@@ -1,66 +1,33 @@
...
 
-:snapshot:`MOVIE: introduction to Blitz Gateway
-<movies/omero-4-3/mov/BlitzGatewayIntro-4.3.mov>`
+To access the OMERO.server Python API, you need to install the Python client
+libraries.
 
+From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
+can be installed via ``pip``. We recommend you use a virtual environment:
+
+    $ python3 -m venv py3_venv
```
Comment (joshmoore) [inline omero/developers/Python.rst:12]: These sections need to be prefaced with:

```
::

    $ python3 ...
```

File: omero/developers/Python.rst [hunk 2]
```diff
@@ -1,66 +1,35 @@
...
+libraries.
 
+From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
+can be installed via ``pip``. We recommend you use a virtual environment:
+
+::
+
+    $ python3 -m venv py3_venv
+    $ source py3_venv/bin/activate
+    $ pip install omero-py
```
Comment (will-moore) [inline omero/developers/Python.rst]: As discussed: need to use the omero-py dev version variable here.

File: omero/conf_autogen.py [hunk 3]
```diff
@@ -13,6 +13,6 @@
 version_dsl_plugin = "5.5.0"
 version_blitz_plugin = "5.5.0"
 version_ice_builder = "1.5.0"
-version_py = "5.5.dev2"
```
Comment (joshmoore) [inline omero/conf_autogen.py:16]: This is handled via the autogen step generally, but bumping manually doesn't hurt. :+1:

**Pull request merged**